kubeflow/pytorch-job/pytorch-job.libsonnet

```diff
@@ -0,0 +1,49 @@
+local k = import "k.libsonnet";
+
+{
+  parts:: {
```

Comment:
The list of components comprising the different prototypes should be listed in the libsonnet file as noted
here
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/new-package-stub/all.libsonnet#L5

This makes it easier to maintain/develop because we don't need to upgrade the prototype just the vendored library